client/src/components/FormInput/FormInputRight.tsx

```diff
@@ -0,0 +1,67 @@
+import { FormControl, InputBase, InputLabel, MenuItem, Select } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+const StyledInput = styled(InputBase)(({ theme }) => ({
+  'label + &': {
+    marginTop: theme.spacing(3),
+  },
+  '& .MuiInputBase-input': {
+    borderRadius: '0px 8px 8px 0px',
+    border: '1px solid #dbdbdb',
+    fontSize: 16,
+    width: '100%',
+    padding: '15px',
+  },
+}));
+
+interface FormInputProps {
+  label: string;
+
+  defaultValue?: any;
+  id: string;
+  select?: boolean;
+  selectOptions?: Array<{ label: string; value: string }> | undefined;
+  selectValue?: string;
+
+  [inputProps: string]: any;
+}
+
+const FormInputRight: React.FC<FormInputProps> = ({
```

Comment:
`FormInputRight` and `FormInputLeft` seem identical, do we need both these components?

Also, the Drop In/Off fields are supposed to be date pickers